qiskit/providers/ibmq/api/rest/root.py

```diff
@@ -279,6 +281,8 @@ def analysis_results(
             params['limit'] = limit
         if marker:
             params['marker'] = marker
+        if verified:
```

Comment:
`verified` is a boolean so if you wanted to filter on `verified=False` this condition would not work.
```suggestion
        if verified is not None:
```